src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -121,9 +125,14 @@ private Task DoInitialFetch()
             int retryCount = 0;
             while (!this.sonarQubeService.IsConnected && retryCount < 30)
             {
+                if (this.initialFetchCancellationTokenSource.IsCancellationRequested)
+                {
+                    return Task.CompletedTask;
```

Comment:
The normal pattern is to throw here, but we know the task was triggered internally by this class so we'll just end (otherwise, we'd have to put a try-catch around the places that _wait_ for the task).